extensions/ui/org.eclipse.smarthome.ui.paper/README.md

```diff
@@ -18,6 +18,18 @@ The following features are implemented:
 * Configuration of services
 * Event support for item state updates, Thing status updates and new inbox entries
 
+## Development
```

Comment:
Sorry, I didn't look at this earlier.
Please note that the README.md is the end user documentation and not meant for developers - you should thus add this section rather to https://github.com/eclipse/smarthome/blob/master/docs/documentation/development/notes.md#paperui-development-jshtml.